docs/08-tips-and-tricks.md

```diff
@@ -0,0 +1,121 @@
+# Tips and tricks
+This section explains some tips and tricks to familiarize with OpenShift 4.
+
+## Ignition files
+
+You can use `jq` to read the files easily:
+
+```
+$ jq < master.ign
+{
+  "ignition": {
+    "config": {
+      "append": [
+        ...
+  "storage": {},
+  "systemd": {}
+}
+```
+
+## NTP configuration
+
+In order to configure NTP, it is required to use the `machine-config-operator`
+to modify the files used by the masters and workers.
+
+RHCOS uses `chrony` to synchronize the time, so it is needed to modify the
+`/etc/chrony.conf` file.
+
+* Create the proper file with your custom tweaks and encode it as base64:
+
+```
+cat << EOF | base64
+server clock.redhat.com iburst
+driftfile /var/lib/chrony/drift
+makestep 1.0 3
+rtcsync
+logdir /var/log/chrony
+EOF
+```
+
+* Create the MachineConfig files as:
+
+```
+cat << EOF > ./masters-chrony-configuration.yaml
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: masters-chrony-configuration
+spec:
+  config:
+    ignition:
+      config: {}
+      security:
+        tls: {}
+      timeouts: {}
+      version: 2.2.0
+    networkd: {}
+    passwd: {}
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,c2VydmVyIGNsb2NrLnJlZGhhdC5jb20gaWJ1cnN0CmRyaWZ0ZmlsZSAvdmFyL2xpYi9jaHJvbnkvZHJpZnQKbWFrZXN0ZXAgMS4wIDMKcnRjc3luYwpsb2dkaXIgL3Zhci9sb2cvY2hyb255Cg==
+          verification: {}
+        filesystem: root
+        mode: 420
+        path: /etc/chrony.conf
+  osImageURL: ""
+EOF
+```
+
+Substitute the base64 string with your own.
+
+* Apply it
+
+```
+oc apply -f ./masters-chrony-configuration.yaml
+```
+
+## OCP Master configuration
+The master configuration is now stored in a `configMap`. During the installation
+process, a few `configMaps` are created, so in order to get the latest:
+
+```
+oc get cm -n openshift-kube-apiserver | grep config
+```
+
+Observe the latest id and then:
+
+```
+oc get cm -n openshift-kube-apiserver config-ID
+```
+
+To get the output in a human-readable form, use:
+
+```
+oc get cm -n openshift-kube-apiserver config-ID -o jsonpath='{.data.config\.yaml}' | jq
+```
+
+For the OpenShift api configuration:
+
+```
+oc get cm -n openshift-apiserver config -o jsonpath='{.data.config\.yaml}' | jq
+```
+
+## Delete 'Completed' pods
```

Comment:
Why?

Reply:
Why not? :smiley:

Reply:
If you're going to have a tips without explaining why it's pretty awkward.

If there's not a good reason to do it, why do it?

Reply:
I've added some clarification.